app/controllers/conversations_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ConversationsController < ApplicationController
+  before_action :authenticate_user
```

Comment:
Nice! Is the conversation controller the only place you want to protect your routes?